ci/playbooks/templates/deployment.yaml.j2

```diff
@@ -113,14 +113,6 @@ kubernetes:
   - Multus_network:
     - CNI:
       - dhcp
```

Comment:
Please keep these entries in for standard CI. We can add in logic for removing these entries with J2 for running the conformance tests if you wish

Reply:
Sure, I will add them back after making sure they work with k8s 1.15 or above.